src/errors/validatorErrors.ts

```diff
@@ -26,6 +31,9 @@ export class InvalidDocument extends PushApiClientBaseError {
       ].join('\n')
     );
   }
+  public get document(): Readonly<Record<string, PrimitivesValues>> {
```

Comment:
Not sure if the invalid document should be part of the error message or if it should be returned separately